client/src/main/scala/drt/client/components/FlightTable.scala

```diff
@@ -48,6 +48,7 @@ object FlightTable {
                    flightManifestSummaries: Map[ManifestKey, FlightManifestSummary],
                    arrivalSources: Option[(UniqueArrival, Pot[List[Option[FeedSourceArrival]]])],
                    originMapper: (PortCode, Option[PortCode], html_<^.TagMod) => VdomNode,
+                   hidePaxDataSourceDescription: Option[Boolean],
```

Comment:
We don't need to pass an Option around as it just means the logic for a default has to be spread to wherever it's used. Keep the logic for the default at the first place we store it